nvtabular/inference/graph/ops/milvus.py

```diff
@@ -0,0 +1,63 @@
+# import json
+
+import numpy as np
+from pymilvus import (  # utility,     <-- useful for dropping the collection
+    Collection,
+    CollectionSchema,
+    DataType,
+    FieldSchema,
+    connections,
+)
+
+from nvtabular.inference.graph.ops.operator import InferenceDataFrame, PipelineableInferenceOperator
+
+
+class QueryMilvus(PipelineableInferenceOperator):
```

Comment:
remove whole file, operating not fully working yet, should not be in code in current state.